index.d.ts

```diff
@@ -7,30 +8,63 @@ declare module 'react-native-picker-select' {
         key?: string | number;
         color?: string;
     }
-    export interface PickerProps {
+    export interface PickerStyle {
+        chevron?: ViewStyle;
+        chevronActive?: ViewStyle;
+        chevronContainer?: ViewStyle;
+        chevronDown?: ViewStyle;
+        chevronUp?: ViewStyle;
+        done?: TextStyle;
+        headlessAndroidContainer?: ViewStyle;
+        headlessAndroidPicker?: ViewStyle;
+        iconContainer?: ViewStyle;
+        inputAndroid?: TextStyle;
+        inputAndroidContainer?: ViewStyle;
+        inputIOS?: TextStyle;
+        inputIOSContainer?: ViewStyle;
+        modalViewBottom?: ViewStyle;
+        modalViewMiddle?: ViewStyle;
+        modalViewTop?: ViewStyle;
+        placeholder?: TextStyle;
+        viewContainer?: ViewStyle;
+    }
+
+    // Omit props needed by the library
+    type PickerModalProps = Omit<
+        ModalProps,
+        | 'testID'
+        | 'visible'
+        | 'transparent'
+        | 'animationType'
+        | 'supportedOrientations'
+        | 'onDismiss'
+        | 'onOrientationChange'
+    >;
+
+    export interface PickerSelectProps {
```

Comment:
This is a breaking change to avoid name clash with React Native `PickerProps`

Reply:
Another alternative is to change the import to `PickerProps as RNPickerProps` and keep the old naming scheme